src/components/dms/dialogs/TextInput.tsx

```diff
@@ -1 +0,0 @@
-export {BottomSheetTextInput as TextInput} from '@discord/bottom-sheet/src'
```

Comment:
This had keyboard handling built in, at least for the original sheet implementation. Unsure of Discord's.